website/docs/backends/config.html.md

```diff
@@ -74,7 +74,7 @@ There are several ways to supply the remaining arguments:
     `init` command line. Note that many shells retain command-line flags in a
     history file, so this isn't recommended for secrets. To specify a single
     key/value pair, use the `-backend-config="KEY=VALUE"` option when running
-    `terraform init`.
+    `terraform init`. For a block, this can be: `-backend-config 'HASH=[{KEY="VALUE"}]'`
```

Comment:
Good one! Guess this format is a bit nasty as `-backend-config 'HASH={KEY="VALUE"}'` would be more inline with what a user would expect (IMHO).

Did you test this with TF v0.12, or only with TF v0.11.x?

Reply:
Only with v0.11.10

Reply:
Just with 0.11, let me try with 0.12 also 👍

Reply:
"Hash" isn't a term we use in Terraform. Since it seems like we're talking about blocks here -- the `workspaces` block in the remote backend, I assume? -- the appropriate noun would be "object".

Part of me wants to instead change this to recommend using this form only for primitive types and to use a separate file for anything more complex, since this approach of packing everything into a string on the CLI is pretty unreadable and error-prone if people are inserting variables that might contain HCL special characters, etc. :thinking: